components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.h

```diff
@@ -60,10 +60,17 @@
 @property(nonatomic, assign) BOOL addLeadingPadding;
 
 /**
- The inset or outset margins for the rectangle surrounding all of the button’s content.
+ The inset or outset margins for the rectangle surrounding all of the cell's content.
 
  Defaults to @c UIEdgeInsetsZero.
  */
 @property(nonatomic, assign) UIEdgeInsets contentEdgeInsets;
 
+/**
+ The inset or outset margins for the rectangle surrounding all of the cell's content.
```

Comment:
Shouldn't this be the cell's image? Both of the documentation are the `cell's content`.

Reply:
Yep, good catch. Copy-paste error.

Reply:
Done.